.travis.yml

```diff
@@ -1,31 +1,32 @@
 language: php
 
-dist: precise
-
 php:
   - 5.5
   - 5.6
   - 7.0
   - 7.1
-  - nightly
+  - 7.2
   - hhvm
-
-sudo: false
-
+  - nightly
 env:
   - COMPOSER_OPTS=""
   - COMPOSER_OPTS="--prefer-lowest"
 
 matrix:
   allow_failures:
+    - php: 7.2
+      env: COMPOSER_OPTS="--prefer-lowest"
```

Comment:
Is this scenario only failing because of incompatibilities in PHPUnit? They don't officially support PHP 7.2 yet, so it makes sense not to treat this as a release blocker.

This shouldn't be a blocker for this change, but in a follow-up PR it might be worth upgrading the tests to PHPUnit's namespaced class style so that you can also allow newer environments to run PHP 6.

Reply:
There are a few requirements that look like they will need their floors raised. I've added this to the backlog to take care of before 7.2 official release, as well as looking at updating our test suite.